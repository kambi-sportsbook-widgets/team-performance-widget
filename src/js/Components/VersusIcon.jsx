import React from 'react';
import PropTypes from 'prop-types';

const VersusIcon = () => {
   return <div className='kw-versusicon' />;
};

export default VersusIcon;
