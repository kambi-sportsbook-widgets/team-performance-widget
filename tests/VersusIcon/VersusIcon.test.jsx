/* eslint-env jest */
import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';
import VersusIcon from '../../src/js/Components/VersusIcon';

let renderer;

describe('VersusIcon DOM rendering', () => {
   beforeEach(() => {
      renderer = ReactTestUtils.createRenderer();
   });
   it('renders correctly', () => {
      expect(renderer.render(
         <VersusIcon />
      )).toMatchSnapshot();
   });
});
