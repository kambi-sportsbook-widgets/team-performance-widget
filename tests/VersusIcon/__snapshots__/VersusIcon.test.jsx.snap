// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VersusIcon DOM rendering renders correctly 1`] = `
<div
  className="kw-versusicon"
/>
`;
